### PR TITLE
Port-8073 | PORT-8128 | Updated action guide for creating pagerduty incident

### PR DESCRIPTION
**Pull request by mk-armah**
# Description

-  Changed action type from CREATE to DAY-2.
-  Updated port action to send payload context directly as inputs.
-  Updated GitHub action to utilize inputs from port context.

## Updated docs pages

Path for the updated docs

/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-incident

### CODE REVIEW

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-incident.md [hunk 1]
```diff
@@ -130,47 +138,24 @@ Create a new self service action using the following JSON configuration.
...
-          "properties": {
-            "{{if (.inputs | has(\"title\")) then \"title\" else null end}}": "{{.inputs.\"title\"}}",
-            "{{if (.inputs | has(\"extra_details\")) then \"extra_details\" else null end}}": "{{.inputs.\"extra_details\"}}",
-            "{{if (.inputs | has(\"urgency\")) then \"urgency\" else null end}}": "{{.inputs.\"urgency\"}}",
-            "{{if (.inputs | has(\"from\")) then \"from\" else null end}}": "{{.inputs.\"from\"}}"
-          },
-          "censoredProperties": "{{.action.encryptedProperties}}"
-        }
+      "{{if (.inputs | has(\"title\")) then \"title\" else null end}}": "{{.inputs.\"title\"}}",
+      "{{if (.inputs | has(\"extra_details\")) then \"extra_details\" else null end}}": "{{.inputs.\"extra_details\"}}",
```
Comment (pazhersh) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-incident.md:151]:
```suggestion
      "extra_details": "{{.inputs.\"extra_details\"}}",
```

Comment (pazhersh) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-incident.md:151]: This goes for the other inputs as well

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-incident.md [hunk 2]
```diff
@@ -130,47 +138,24 @@ Create a new self service action using the following JSON configuration.
...
-            "{{if (.inputs | has(\"extra_details\")) then \"extra_details\" else null end}}": "{{.inputs.\"extra_details\"}}",
-            "{{if (.inputs | has(\"urgency\")) then \"urgency\" else null end}}": "{{.inputs.\"urgency\"}}",
-            "{{if (.inputs | has(\"from\")) then \"from\" else null end}}": "{{.inputs.\"from\"}}"
-          },
-          "censoredProperties": "{{.action.encryptedProperties}}"
-        }
+      "{{if (.inputs | has(\"title\")) then \"title\" else null end}}": "{{.inputs.\"title\"}}",
+      "extra_details": "{{.inputs.\"extra_details\"}}",
+      "{{if (.inputs | has(\"urgency\")) then \"urgency\" else null end}}": "{{.inputs.\"urgency\"}}",
+      "{{if (.inputs | has(\"from\")) then \"from\" else null end}}": "{{.inputs.\"from\"}}",
```
Comment (pazhersh) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/PagerDuty/create-pagerduty-incident.md:153]:
```suggestion
      "from": "{{.inputs."from"}}",
```
and the other inputs too

**Review by yairsimantov20 — CHANGES_REQUESTED** [review]
go over @pazhersh comments

**Review by pazhersh — APPROVED** [review]

**Pull request merged**